### ISSUE TITLE
NoMe-Seq data

### ISSUE THREAD

**Issue by vivekbhr** [adamewing/methylartist#31]
Hi @adamewing 

I am trying to plot the GpC methylation from modBAM format file (with ML and `MM:Z:C+m` tags) using the following command:

```
methylartist locus --motif C \
--motifsize 1 \
-i chr9:37137484-38116750 \
-g data/gtf.gz \
--ref genome.fa \
-b mod_mappings.bam
```

But I keep getting the error: `insufficient coverage to plot` . I tried other variants like: `--motif G[C]H --motifsize 3` or `--motif CH` , but get the same error. Would you have input on how I can make it work?

Thanks,
Vivek

**Comment by adamewing** [comment]
Hi Vivek, I would have guessed NOMe-seq via M. CviPI (GpC methylation) had a "GC" motif but if it's tagged with `MM:Z:C+m` then `--motif C` should work too. Silly question but to start with troubleshooting, does `chr9:37137484-38116750` have reads that overlap it?

**Comment by vivekbhr** [comment]
Indeed, samtools view suggests about 2000 reads on that region. I attached a few reads from samtools view output from the region:.
[test.txt](https://github.com/adamewing/methylartist/files/10085394/test.txt)

**Comment by vivekbhr** [comment]
Hi @adamewing any tips on how to make it work?

**Comment by vivekbhr** [comment]
Thanks. I can send you a transfer link for a small example BAM file if you let me know your email.

**Comment by adamewing** [comment]
Hi, sure you can send the link to adam.ewing@gmail.com, thanks.
